web-console/src/app/(root)/(blank)/login/page.tsx

```diff
@@ -0,0 +1,18 @@
+'use client'
```

Comment:
how does the login page look like? we should make sure it's consistent with the layout of console